x-pack/plugins/cloud_security_posture/public/test/fixtures/resource_findings_fixture.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EcsEvent } from '@kbn/ecs';
+import Chance from 'chance';
+import { CspFinding } from '../../../common/schemas/csp_finding';
+
+const chance = new Chance();
+
+export const getResourceFindingsTableFixture = (): CspFinding & { id: string } => ({
```

Comment:
can we rename to `getFindingsFixture` ? as i don't think the data is specific for a single table?

also it'd probably be nice, not critical for this PR tho, if we could remove similar code from other tests and re-use this fixture across all findings tests. (there is also a fixture in `__mocks__` for the findings flyoyt),

Reply:
I'll address these changes in a chore clean-up PR.